src/LearningObjectStats/LearningObjectStatStore.ts

```diff
@@ -36,7 +38,25 @@ export class LearningObjectStatStore implements LearningObjectStatDatastore {
         unit: 0,
         course: 0,
       },
+      blooms_distribution: {
+        apply: 0,
+        evaluate: 0, 
+        remember: 0,
+      }
     };
+    for(var i=0; i<bloomsData.length; i++){
+      if(bloomsData[i]._id === 'Apply and Analyze'){
```

Comment:
For maintainability, create a `const` to store the string ids of each bloom that can be easily modified and accessed. For instance 
```suggestion
const BLOOMS = {
APPLY: 'Apply and Analyze'
...
}
      if(bloomsData[i]._id === BLOOMS.APPLY){
```

Reply:
Also, since we have defined cases here of what we want to match, a switch case statement may be more appropriate and easier to read and maintain.